packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -94,6 +99,16 @@ export class InsuredBridgeL1Client {
     return this.relays[l1Token][deposit.depositHash];
   }
 
+  hasInstantRelayer(l1Token: string, depositHash: string, realizedLpFeePct: string): boolean {
```

Comment:
nit: why are you taking in `realizedLpFeePct`? it might be better to pass this around as a `BN`, even though you are casing it to a hash later on. Reasoning is mainly just to maintain consistency with the rest of numbers in this client, implementation and other downstream usage of `Pct` variables.

Reply:
I was matching the type of `realizedLpFeePct` in the `Relay` struct isn't that more consistent?

Reply:
everything in that struct is a string to make the storage cheaper in memory.

Reply:
otherwise, what you have will also be fine. it's not too different to how other params are setup